### PR TITLE
Adding lockdoor framework

### PR DESCRIPTION
**Pull request by SofianeHamlaoui**
LockDoor is a Framework aimed at helping penetration testers, bug bounty hunters And cyber security engineers. 
This tool is designed for Debian/Ubuntu/ArchLinux based distributions to create a similar and familiar distribution for Penetration Testing. 
But containing the favorite and the most used tools by Pentesters. 
As pentesters, most of us has his personal ' /pentest/ ' directory so this Framework is helping you to build a perfect one. 
With all of that ! It automates the Pentesting process to help you do the job more quickly and easily.
Link : https://lockdoor.sofianehamlaoui.me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ A curated list of awesome things and projects built by Algerian developers.
   - [Facial-expression-recognition-svm](https://github.com/amineHorseman/facial-expression-recognition-svm) - Training SVM classifier to recognize facial expressions (emotions)
   - [Malware Revealer](https://github.com/malware-revealer/malware-revealer) - A malware classification framework, designed primarily for malware detection, it contains a modular toolset for feature extraction, as well as pre-trained models and a ready to use web API for making predictions.
   - [Tchamba.random](https://github.com/Fcmam5/tchamba) package of functions generating random data (colors, names, jokes, letters..)
+  - [Lockdoor Framework ](https://github.com/SofianeHamlaoui/Lockdoor-Framework) -  A Penetration Testing framework with Cyber Security Resources
```
Comment (youknowriad) [inline README.md:59]: Can we put it just before "Malware Revealer" (alphabetical order)? Thanks

Comment (SofianeHamlaoui) [reply README.md:59]: As you like, I just thought that we had to add them by (date of commit).
But sure, why not 😄

Comment (youknowriad) [reply README.md:59]: I'm actually waiting for the reordering before merging, as you can see, everything is already alphabetically ordered in the file (per section)

Comment (SofianeHamlaoui) [reply README.md:59]: Aw perfect, I'll do a new PR after reordering them

**Pull request closed without merging**